backend/tournesol/models/scaling.py

```diff
@@ -0,0 +1,54 @@
+"""
+Models related to scaling values, used to calibrate individual scores in order
+to aggregate them in a Byzantine-resilient manner.
+"""
+
+from django.db import models
+
+from core.models import User
+
+from .poll import Poll
+
+
+class ContributorScaling(models.Model):
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="scalings",
+    )
+    poll = models.ForeignKey(
+        Poll,
+        on_delete=models.CASCADE,
+        related_name="contributor_scalings"
+    )
+    criteria = models.TextField(
```

Comment:
1. I wanted to use the same field definition as other models (`EntityCriteriaScore`, etc.) which were defined before the `Criteria` model had been created.
2. I don't think there would be an immediate benefit to use a ForeignKey to `Criteria`, as no other property is used except its `name`.
3. If we are not careful, using a `ForeignKey` in existing models could cause a significant performance hit as any access to `.criteria` would cause an additional database query per object.
4. Migrating `criteria` to a `ForeignKey` in all models may not be trivial. The easiest approach would be to define the "name" as the primary key for `Criteria` (no migration needed in other models data). But then, it would not be possible to modify the name of an existing criteria, and that would defeat some of the benefit of migrating. The more usual approach would be to replace  the criteria name in all models with an opaque id used as a foreign key. This kind of migration is probably harder to implement, and then accessing the criteria name would require an extra join (e.g via `.select_related()`) compared to the current implementation.

In the end, I don't think it's worth the complication right now.